pkg/collector/corechecks/net/wlan/wlan_darwin.go

```diff
@@ -50,7 +82,7 @@ func GetWiFiInfo() (WiFiInfo, error) {
 		Noise:           int(info.noise),
 		TransmitRate:    float64(info.transmitRate),
 		HardwareAddress: hardwareAddress,
-		ActivePHYMode:   int(info.activePHYMode),
+		PhyMode:         formatPhy(phyMode(info.activePHYMode)),
```

Comment:
```suggestion
		PhyMode:         phyMode(info.activePHYMode),
```
This should probably work if we define the String() method.
if not, we can just call string().

Reply:
Thank you, I cannot believe I missed that (plus I did not check CLI results). Addressed.